conf/messages.en

```diff
@@ -1439,3 +1439,16 @@ fhlIncomeIntro.details.l1 = rental income
 fhlIncomeIntro.content.para2.individual = If your properties are jointly owned, only tell us about your share of these figures.
 fhlIncomeIntro.content.para2.agent = If your client's properties are jointly owned, only tell us about their share of these figures.
 
+totalIncomeAmount.title.individual=How much total rent a room income did you receive?
```

Comment:
Need to distinguish the journey here I think

Reply:
`ukrentaroom.income.totalIncomeAmount.title.individual=How much total rent a room income did you receive?`